CONTRIBUTING.md

```
@@ -322,6 +353,51 @@
+>[!WARNING]  
+> After your code is merged into the `stage` branch, you should **not** run the bot locally. All testing at this point must be performed using the staging bot instance.
+
+#### How to Test in Staging
+
+1. **Push your branch to `stage`:**  
+  Merge your feature branch into the `stage` branch following the standard workflow.
+
+2. **Deployment:**  
+  The bot will be automatically deployed to the Staging environment via our CI/CD pipeline.
+
+3. **Testing:**  
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)